operator/config/prometheus/pa.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: security.istio.io/v1beta1
```

Comment:
this is an istio resource in prometheus we should avoid this if possible and create another baes / overlay